etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/EtorDomainRegistration.java

```diff
@@ -231,6 +242,34 @@ DomainResponse handleMetadata(DomainRequest request) {
         }
     }
 
+    DomainResponse handleResults(DomainRequest request) {
+        Result<?> results;
+
+        String receivedSubmissionId = request.getHeaders().get("recordid");
+        if (receivedSubmissionId == null || receivedSubmissionId.isEmpty()) {
+            receivedSubmissionId = null;
+            logger.logError("Missing required header or empty: RecordId");
+        }
+
+        String errorMessage = "";
+        try {
+            results = resultController.parseResults(request);
+            sendResultUseCase.convertAndSend(results, receivedSubmissionId);
+        } catch (FhirParseException e) {
+            errorMessage = "Unable to parse result request";
```

Comment:
If the error message string is only used in place, I don't think a variable would be needed. Same for the one in the other catch